java-maven/maven-perforce/Jenkinsfile

```diff
@@ -0,0 +1,9 @@
+regressionTest {
```

Comment:
you really think its important - it will consume time and resources.....

Reply:
you're totally right